### ISSUE TITLE
Using debug console picker when not paused leaves console unresponsive

### ISSUE THREAD

**Issue by roblourens** [microsoft/vscode#152407]
- Start two node launch configs
- Both sessions are running and not paused
- Use the debug console switcher to switch consoles
- Evaluate `1+1`, there is no response

In the console picker, we show parent sessions. The child needs to be focused to get the evaluate requests. When a child is paused, we will focus that one with this code

https://github.com/microsoft/vscode/blob/16c2a3ab3be4cf89c71fcce1ff1c11a06fcb923a/src/vs/workbench/contrib/debug/browser/repl.ts#L890-L896

I think this is the same as https://github.com/microsoft/vscode/issues/112595 except we still need to pick a child when not paused, and it also doesn't work when a deeper child is stopped instead.

https://github.com/microsoft/vscode/issues/138963 is relevant too

Basically when something tries to focus a new wrapper parent session, we should always focus one of its children instead. cc @isidorn

**Comment by isidorn** [comment]
@roblourens agree. It feels very similar to that one.
